dynamics-nav/Delete_-part--Operation.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: "Delete_&lt;part&gt; Operation"
```

Comment:
The title here is messing up I think due to the "<" ">" parts. I assume we can't touch the titles, though?